<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Добавить фильм</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Название фильма*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="film_name">
       <property name="font">
        <font>
         <family>MS Reference Sans Serif</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Изображение*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="picture_film">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="load_pic_btn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Загрузить изображение</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>О фильме*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="about_of_film"/>
     </item>
     <item>
      <widget class="QPushButton" name="add_btn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
